package.json

```diff
@@ -257,6 +267,16 @@
           "command": "quarkusTools.deployToOpenShift.short",
           "when": "quarkusProjectExistsOrLightWeight && resourceExtname == .java",
           "group": "navigation@10"
+        },
+        {
+          "command": "qute.validation.enabled.toggle.off",
+          "when": "quarkusProjectExistsOrLightWeight && editorLangIdSupportsQute && quteValidationEnabled",
```

Comment:
I  will rename quteValidationEnabled with editorQuteValidationEnabled

@rgrunber what do you think?

Reply:
quteValidationEnabled(i.e. qute.validation.enabled) is the global setting, not the file specific setting, I feel editor in the name is unnecessary

Reply:
quteValidationEnabled is the validation state for a given qute template (for a given editor), like editorLangIdSupportsQute,so I think we should starts this property with `editor` prefix